docs/api/app.md

```diff
@@ -1426,6 +1440,28 @@ details.
 
 **Note:** Enable `Secure Keyboard Entry` only when it is needed and disable it when it is no longer needed.
 
+### `app.isActive()` _macOS_
+
+Returns `boolean` - `true` if the app is currently active (i.e., the macOS
+title bar is showing the name of the app).
+
+See [`-[NSApplication active]`](https://developer.apple.com/documentation/appkit/nsapplication/1428493-active?language=objc).
+
+### `app.activate([opts])` _macOS_
+
+* `opts` Object (optional)
+  * `ignoreOtherApps` boolean - If false, the app is activated only if no other
+    app is currently active. If true, the app activates regardless. Defaults to
+    false.
+
+See [`-[NSApplication activateIgnoringOtherApps:]`](https://developer.apple.com/documentation/appkit/nsapplication/1428468-activateignoringotherapps?language=objc).
+
+### `app.deactivate()` _macOS_
+
+Deactivates the app.
```

Comment:
Given 
> Normally, you shouldn’t invoke this method—AppKit is responsible for proper deactivation.

from official docs and the fact that this method is (as you note) unclear in terms of when it should be invoked and why, I don't think we should expose this directly. Without clear guidelines as to usage, I think it'll be more confusing than beneficial.

Reply:
I suppose that's fair enough. I was erring on the side of including it on the off-chance that there was a good reason to invoke it in some cases, & that others might have a clearer idea of when that might be. & for such cases, it would be convenient if this already existed! But probably it's more confusing than it's worth.